packages/react-query/src/scripts/utils/package.ts

```diff
@@ -49,3 +51,41 @@ export function getSuspensiveReactQueryPackageJson(targetVersion: string) {
 
   return packageJson
 }
+
+export function getIndexFileContent() {
+  const basePath = path.resolve(__dirname, '../../dist').replace(/src/, '')
+
+  return fs.readFileSync(path.join(basePath, 'index.js'), 'utf-8') || ''
+}
```

Comment:
(Priority low) In my opinion we could __dirname as parameter to fix start position regardless of getIndexFileContent's executing position.

```suggestion
export function getIndexFileContent(dirname: string) {
  const basePath = path.resolve(dirname, 'dist').replace(/src/, '')

  return fs.readFileSync(path.join(basePath, 'index.js'), 'utf-8') || ''
}
```